<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainBox_GUI</class>
 <widget class="QWidget" name="MainBox_GUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>420</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="main_frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>7</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/1.png</normaloff>:/red/1.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 1</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="Controls_widget" name="controls_widget" native="true"/>
          </item>
          <item>
           <widget class="Check_in_widget" name="check_in_widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/2.png</normaloff>:/red/2.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 2</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QToolBox" name="toolBox">
            <property name="currentIndex">
             <number>3</number>
            </property>
            <widget class="QWidget" name="page">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/black/1.png</normaloff>:/black/1.png</iconset>
             </attribute>
             <attribute name="label">
              <string>Page 1</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_2">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/black/2.png</normaloff>:/black/2.png</iconset>
             </attribute>
             <attribute name="label">
              <string>Page 2</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_3">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>98</width>
               <height>28</height>
              </rect>
             </property>
             <attribute name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/black/3.png</normaloff>:/black/3.png</iconset>
             </attribute>
             <attribute name="label">
              <string>Page 3</string>
             </attribute>
            </widget>
            <widget class="QWidget" name="page_4">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>578</width>
               <height>169</height>
              </rect>
             </property>
             <attribute name="icon">
              <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
               <normaloff>:/black/4.png</normaloff>:/black/4.png</iconset>
             </attribute>
             <attribute name="label">
              <string>Page 4</string>
             </attribute>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/3.png</normaloff>:/red/3.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 3</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="File_widget" name="file_widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/4.png</normaloff>:/red/4.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 4</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="Table_widget" name="table_widget"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/5.png</normaloff>:/red/5.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 5</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="MDI_widget" name="mdi_widget" native="true"/>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/6.png</normaloff>:/red/6.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 6</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCalendarWidget" name="cw_calendar"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_show">
              <property name="text">
               <string>show</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>292</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_7">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/7.png</normaloff>:/red/7.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 7</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QTreeWidget" name="treeWidget">
            <column>
             <property name="text">
              <string>Root</string>
             </property>
            </column>
            <item>
             <property name="text">
              <string>Element root</string>
             </property>
             <item>
              <property name="text">
               <string>Element 1</string>
              </property>
              <property name="checkState">
               <enum>Unchecked</enum>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Element 2</string>
              </property>
              <property name="checkState">
               <enum>Unchecked</enum>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Element 3</string>
              </property>
              <property name="checkState">
               <enum>Unchecked</enum>
              </property>
             </item>
            </item>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="btn_1">
              <property name="text">
               <string>1</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_2">
              <property name="text">
               <string>2</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_8">
         <attribute name="icon">
          <iconset resource="../../../../../../lib2/icons/digits/digits.qrc">
           <normaloff>:/red/8.png</normaloff>:/red/8.png</iconset>
         </attribute>
         <attribute name="title">
          <string>Tab 8</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QPushButton" name="pushButton">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="pushButton_5">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="pushButton_7">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QPushButton" name="pushButton_9">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QPushButton" name="pushButton_6">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="pushButton_8">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="pushButton_4">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item row="0" column="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QTextEdit" name="textEdit"/>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QPlainTextEdit" name="plainTextEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="test_frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QSpinBox" name="sb_test"/>
         </item>
         <item>
          <widget class="QLineEdit" name="le_test"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Table_widget</class>
   <extends>QTableWidget</extends>
   <header>table_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Check_in_widget</class>
   <extends>QWidget</extends>
   <header>check_in_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MDI_widget</class>
   <extends>QWidget</extends>
   <header>mdi_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>File_widget</class>
   <extends>QWidget</extends>
   <header>file_widget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Controls_widget</class>
   <extends>QWidget</extends>
   <header>controls_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../lib2/icons/digits/digits.qrc"/>
 </resources>
 <connections/>
</ui>
